src/components/UpcomingEventCard.vue

```diff
@@ -224,24 +267,78 @@ export default {
 	.text {
 		font-size: 0.75rem;
 		font-weight: 500;
-		line-height: 0.87rem;
 	}
 
 	.just-sb {
 		justify-content: space-between;
 	}
 
 	.link {
-		text-decoration: underline;
+		padding-left: 1.25rem;
+	}
+
+	.talk-link {
+		padding-left: 1rem;
```

Comment:
Padding should be `1.25rem` like the other links, and this one should be underlined. The colour should also be different according to the figma.